storio-contentresolver/src/main/java/com/pushtorefresh/storio/contentresolver/query/DeleteQuery.java

```diff
@@ -96,33 +94,15 @@ public CompleteBuilder uri(@NonNull Uri uri) {
         }
 
         /**
-         * Optional: Specifies where clause
-         * <p>
-         * Default value is <code>null</code>
-         *
-         * @param where optional restriction to apply to rows when deleting
-         * @return builder
-         */
-        @NonNull
-        public Builder where(@Nullable String where) {
-            this.where = where;
-            return this;
-        }
-
-        /**
-         * Optional: Specifies arguments for where clause
-         * <p>
-         * Passed objects will be immediately converted to list of {@link String} via calling {@link Object#toString()}
-         * <p>
-         * Default value is <code>null</code>
+         * Required: Specifies uri
          *
-         * @param whereArgs arguments for {@link DeleteQuery#where}
+         * @param uri full URI to query, including a row ID (if a specific record is requested)
          * @return builder
          */
         @NonNull
-        public Builder whereArgs(@Nullable Object... whereArgs) {
-            this.whereArgs = QueryUtil.varargsToList(whereArgs);
-            return this;
+        public CompleteBuilder uri(@NonNull String uri) {
+            this.uri = Uri.parse(uri);
+            return new CompleteBuilder(this);
```

Comment:
Same question, as in previous PR. 
I think we need override `uri` setter or just copy all fields in `CompleteBuilder` constructor.